cubes/query/browser.py

```diff
@@ -75,6 +77,41 @@
 _ReportResult = Union[AggregationResult, Facts, JSONType, List[JSONType]] 
 
 
+class BrowserFeatureAction(Enum):
+    aggregate = 1
+    fact = 2
+    facts = 3
+    members = 4
+    cell = 5
+
+    @classmethod
+    def from_string(cls, actions: Iterable[str]) -> List['BrowserFeatureAction']:
+        if not actions:
+            return []
+
+        return [cls[action] for action in actions if action in cls.__members__.keys()]
+
+
+class _BaseBrowserFeatures(NamedTuple):
+    actions: List[BrowserFeatureAction]
+    aggregate_functions: List[str]
+    post_aggregate_functions: List[str]
+
+
+class BrowserFeatures(_BaseBrowserFeatures):
+    @property
+    def asdict(self) -> JSONType:
```

Comment:
Missed that in the documentation about NamedTuple, but it makes sense that there is such protection.

What do you think about making it a cheap class with `to_dict() → JSONType` and classmethod `from_dict(JSONType) → BrowserFeatures` (btw. this to_dict/from_dict is going to be pattern with all exportable objects).